consensus/src/pos/error.rs

```diff
@@ -60,4 +60,8 @@ pub enum ConsensusPoSError {
     InvalidTargetBlockTime,
     #[error("CRITICAL: Block time must be monotonic")]
     InvariantBrokenNotMonotonicBlockTime,
+
+    // TODO the following error should include the corresponding error from UtxosView
```

Comment:
In those TODOs like this one, can you please reference the PR number so that anyone seeing this would understand the context?

Reply:
Also maybe create an issue and put more details there.

Reply:
Will do